src/ansiblelint/rules/YamllintRule.py

```diff
@@ -48,24 +31,8 @@ class YamllintRule(AnsibleLintRule):
     severity = "VERY_LOW"
     tags = ["formatting", "yaml"]
     version_added = "v5.0.0"
-    config = None
+    config = load_yamllint_config()
     has_dynamic_tags = True
-    if "yamllint.config" in sys.modules:
```

Comment:
This import is no longer conditional, so checking this didn't make sense. I dropped the check while moving the config loading.